platform/android/src/style/sources/source.cpp

```diff
@@ -35,19 +35,6 @@ namespace android {
         return jni::Make<jni::String>(env, source.getID());
     }
 
-    void Source::updateStyle(jni::jboolean updateClasses) {
-        //Update the style only if attached
-        if (ownedSource == nullptr) {
-            Update flags = mbgl::Update::RecalculateStyle;
-            if(updateClasses) {
-                flags = flags | mbgl::Update::Classes;
-            }
-            map->update(flags);
-        } else {
-            mbgl::Log::Debug(mbgl::Event::JNI, "Not updating as source is not attached to map (yet)");
-        }
-    }
```

Comment:
This is removed because it is automatically triggered when updating the GeoJSON data